private/projects/aws-prototyping-sdk-project.ts

```diff
@@ -146,4 +148,25 @@ export class AwsPrototypingSdkProject extends PDKProject {
 
     super.synth();
   }
+
+  /**
+   * @inheritDoc
+   */
+  getNxProjectTargets(): ProjectTargets {
+    const defaultTargets = super.getNxProjectTargets();
+    const relativeDir = `${this.outdir.split(this.root.outdir)[1]}`;
```

Comment:
Should we expose this as a getter on PDKProject? Seems useful

Reply:
Ill keep as is for now as it's an internal construct. If needed we can make public later.